source/agora/consensus/EnrollmentManager.d

```diff
@@ -689,22 +700,25 @@ unittest
     scope storage = new TestUTXOSet;
 
     auto gen_key_pair = WK.Keys.Genesis;
+    UTXOSetValue[Hash] utxos;
```

Comment:
These changes should also be reverted because the utxo set here is not used anymore.